src/front-end/src/components/AdminSignup/AdminSignup.js

```diff
@@ -126,9 +130,13 @@ class AdminSignup extends Component {
                   required: true,
                   message: 'Please confirm your password',
                 },
+                /* Checks whether the input entered in 'confirm password'
+                is the same as what's in the 'password' field
+                */
                 ({ getFieldValue }) => ({
                   validator(rule, value) {
                     if (!value || getFieldValue('password') === value) {
+                      // If the inputs match, remove the red error text
```

Comment:
Since we are already in the if statement, I would just describe what the line below the comment does. Additionally, this comment is incomplete, because there is also the !value that checks if the field is empty. Therefore. I think writing 

```suggestion
                      // Remove the red error text
```

suffices here, with the rest of the content given by the code and your other comment above.